data/bangs.json

```diff
@@ -67393,17 +67393,18 @@
   },
   {
     "s": "Openstreetmap forum Netherlands",
-    "d": "forum.openstreetmap.org",
+    "d": "community.openstreetmap.org",
     "t": "osmforumnl",
-    "u": "https://forum.openstreetmap.org/search.php?action=search&keywords={{{s}}}&forums[]=12",
+    "u": "https://community.openstreetmap.org/search?q={{{s}}}%20%23communities%3Anl",
```

Comment:
The test should really be updated to allow encoded `%23` but this works for now. Thank you.